libdevcore/SwarmHash.h

```diff
@@ -20,12 +20,13 @@
 #pragma once
 
 #include <libdevcore/FixedHash.h>
-#include <libdevcore/Common.h>
+
+#include <string>
```

Comment:
Let's change the coding rule: http://stackoverflow.com/a/614333/5197297 (read the full answer)